doc/Caliptra.md

```diff
@@ -1136,3 +1056,84 @@ This section describes Caliptra error reporting and handling.
 [^8]: ECDSA is used for firmware verification and SPDM (signing).
 
 [^9]: SHA is used with ECDSA and HMAC, and is also used to generate measurements.
+
+# Terminology
+
+The following acronyms and abbreviations are used throughout this document.
+
+| Abbreviation | Description |
+|:-------------|:------------|
```

Comment:
I'd say keep the Terminology section at the beginning. That way folks can peruse the terminology before diving in.

Reply:
readers tend to skip over terminology sections; these sections are for "look-up" vs. "peruse"